src/components/CourseTable/CourseTable.test.jsx

```diff
@@ -4,63 +4,75 @@ import { MemoryRouter } from 'react-router-dom';
 import { Provider } from 'react-redux';
 import configureStore from 'redux-mock-store';
 import thunk from 'redux-thunk';
+import { IntlProvider } from '@edx/frontend-platform/i18n';
 import CourseTable from './index';
+import '@testing-library/jest-dom';
 
 const mockStore = configureStore([thunk]);
-const store = mockStore({});
+const store = mockStore({
+  table: {
+    loading: false,
+    error: null,
+    data: {
+      count: 2,
+      next: null,
+      previous: null,
+      results: [
+        {
+          key: 'hogwarts+123',
+          uuid: '04c512ed-1b2b-461c-a715-00a0f8cba514',
+          title: 'Hast La Vista',
+          course_run_statuses: [
+            'archived',
+          ],
+          editors: [
+            {
+              id: 8,
+              user: {
+                id: 1,
+                full_name: 'edx',
+                email: 'edx@example.com',
+              },
+              course: '04c512ed-1b2b-461c-a715-00a0f8cba514',
+            },
+          ],
+        },
+        {
+          key: 'hogwarts+111123',
+          uuid: '7a6d5b85-fff2-4944-a7bc-a82890de3e7c',
+          title: 'New Era Wisdom',
+          course_run_statuses: [
+            'archived',
+          ],
+          editors: [
+            {
+              id: 17,
+              user: {
+                id: 1,
+                full_name: 'edx',
+                email: 'edx@example.com',
+              },
+              course: '7a6d5b85-fff2-4944-a7bc-a82890de3e7c',
+            },
+          ],
+        },
+      ],
+    },
+  },
+});
 
-describe.skip('CourseTable', () => {
-  // TODO: The table is not rendering properly and fails on the assertion.
+describe('CourseTable', () => {
   it('shows a table', async () => {
     render(
-      <Provider store={store}>
-        <MemoryRouter>
-          <CourseTable />
-        </MemoryRouter>
-      </Provider>,
-    );
-    await waitFor(() => expect(screen.getByRole('table')).toBeInTheDocument());
-  });
-
-  it('hides table and button when blacklisted', async () => {
```

Comment:
[removed tests] These tests were added in [339](https://github.com/openedx/frontend-app-publisher/pull/339) relating to some concept of `org_blacklist` in publisher. The blacklist code was removed in [370](https://github.com/openedx/frontend-app-publisher/pull/370) but the tests remained. In fact, the `hides table and button when blacklisted ` does not "really pass" on [370](https://github.com/openedx/frontend-app-publisher/pull/370) but the failure was missed cuz it was snapshot-based.